Entity.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Entity.h"
+#include <string>
+#include <iostream>
+
+Entity::Entity(const int health,const int attackDamage,const std::string name) : Name(name){
+    this->MaxHealth      =   health;
+    this->Health         =   this->MaxHealth;
+    this->AttackDamage   =   attackDamage;
+    this->isDead         =   false;
+}
+
+int Entity::getHealth(){
+    return this->Health;
+}
+
+int Entity::getAttackDamage(){
+    return this->AttackDamage;
+}
+
+void Entity::die(){
+    this->isDead = true;
+}
```

Comment:
Ha jól látom, ezt egyetlen helyről hívjátok. Nem lenne egyszerűbb ott eleve átállítani ezt a logikai változót?

Reply:
@wajzy  Úgy gondoltuk, hogy logikusabb lenne egy külön "halál" metódus, mert számunkra így átláthatóbb, és ha bármi más dolgot kellene csinálni halálkor, akkor már megvan a szükséges függvényünk hozzá. Amennyiben külön kéri, kivesszük ezt is, viszont számunkra így átláthatóbb.